lib/sul_orcid_client/cocina_support.rb

```diff
@@ -0,0 +1,27 @@
+class SulOrcidClient
+  # Helper methods for working with Orcid in Cocina
+  class CocinaSupport
+    # @param [Cocina::Models::Contributor] contributor to check
+    # @return [Boolean] true unless the contributor has a citation status of false
+    def self.cited?(contributor)
+      contributor.note.none? { |note| note.type == "citation status" && note.value == "false" }
+    end
+
+    # @param [Cocina::Models::Contributor] contributor to check
+    # @return [String, nil] orcid id including host if present
+    def self.orcidid(contributor)
```

Comment:
Am I understanding correctly that this method and `cited_orcids` don't replace any current code in the client, they're being defined for future use by DSA? Since they don't consider ROR identifiers, I'm assuming there is some work to do in DSA (guessing for pushing the work to ORCID?) that will require specifically checking for ORCID.

Reply:
These will be used in DSA and robots (which is why added to this library).